taccsite_cms/templates/cms_menu.html

```diff
@@ -15,7 +15,10 @@
       class="nav-link dropdown-toggle"
       data-toggle="dropdown"
       aria-haspopup="true"
-      aria-expanded="false">
+      aria-expanded="false"
+      role="button"
+      id="navbarDropdown"
```

Comment:
**Bug.** Whoops, I should have caught this in #758. This will make every dropdown have the same ID. Only one element per page may have an ID.

Reply:
The `aria-labelledby="navbarDropdown"`must also be updated. to be unique per dropdown.

Reply:
Ticket: [TUP-662: Nav Dropdown ID Must be Unique](https://tacc-main.atlassian.net/browse/TUP-662)